src/usr/local/php/unraid-tailscale-utils/unraid-tailscale-utils/Info.php

```
@@ -0,0 +1,474 @@
+    }
+
+    private function tr(string $message): string
+    {
+        return $this->tr->tr($message);
+    }
+
+    public function getStatusInfo(): StatusInfo
+    {
+        $status = $this->status;
+        $prefs  = $this->prefs;
+        $lock   = $this->lock;
```

PHPMD (2.15.0)

Avoid unused local variables such as '$lock'. (Unused Code Rules)

(UnusedLocalVariable)